wicketd/src/update_tracker.rs

```diff
@@ -1570,14 +1573,24 @@ impl UpdateContext {
         cx: StepContext<S>,
         stage: ComponentUpdateStage,
         update_id: Uuid,
-        component: &str,
+        component: UpdateComponent,
+        component_name: &str,
     ) -> anyhow::Result<()>
     where
         S::ProgressMetadata: Default,
     {
         // How often we poll MGS for the progress of an update once it starts.
         const STATUS_POLL_FREQ: Duration = Duration::from_millis(300);
 
+        let progress_units = match component {
+            UpdateComponent::Rot => {
+                // XXX what's the correct unit for this?
+                ProgressUnits::new("flash slots scanned")
```

Comment:
@jgallagher what's the correct unit for this?

Reply:
There isn't one - the Rot (currently?) [never](https://github.com/oxidecomputer/hubris/blob/6c0c483c47f5050696032b7166f4b650ca66c993/task/control-plane-agent/src/update/rot.rs#L103-L105) returns a "preparing" status.

Reply:
Made moot by the update to using "preparation steps" unconditionally.